pyproject.toml

```diff
@@ -10,7 +10,17 @@ pandas = "^1.3.5"
 
 [tool.poetry.dev-dependencies]
 flake8 = "^5.0.4"
-isort = "^5.10.1"
+isort = "^5.11.4"
+pytest = "^7.2.1"
+coverage = "^7.0"
+pytest-cov = "^4.0.0"
+chispa = "*"
```

Comment:
chispa not needed as only for pyspark

Reply:
Thanks, David. Chispa removed =) We were trying to keep it consistent with the Statisticial-Methods-Library repo and we were using trying to reuse to code. I guess we didn't end up cleaning up some things. Thanks for the help =)